src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -152,7 +152,7 @@ private void addRandomUserProfiles() {
     List<String> categories = getRandomCategories();
     List<String> subCategories = getRandomSubCategories();
     for (int i = 0; i < DEFAULT_PROFILE_COUNT; i++) {
-      User author = getRandomElement(users);
+      User author = users.get(i); //
```

Comment:
Nice catch that this was causing random users without profiles! I think that with this change you can now use a for-each loop (`for (User user : users) {`) over the users instead, and probably remove the DEFAULT_PROFILE_COUNT variable altogether.